docs/modules/ROOT/pages/how-tos/upgrade.adoc

```diff
@@ -1,16 +1,59 @@
 = How to Upgrade K8up
 
+WARNING: Never remove the CRDs, as you might lose the respective resources!
+
+[#upgrade_1_to_2]
+== Upgrade K8up from 1.x to 2.x
+
+TIP: See also xref:explanations/what-has-changed-in-v2.adoc[].
+
+Only little has changed on from a user's perspective in K8up 2.x, but a lot did change under the hood nonetheless.
```

Comment:
```suggestion
Because of the move of K8up to it's own GitHub organization, and a cleanup of several namings, there are visible changes from a user's perspective in K8up 2.x, which require (manual) intervention for a successful upgrade.
```